### ISSUE TITLE
[Accessibility][DetailsList][Keyboard]: Unable to move focus one cell up/down using up/down keys

### ISSUE THREAD

**Issue by satishc2437** [OfficeDev/office-ui-fabric-react#1057]
Hi,
We are using the DetailsList control to create a Grid that will have input text boxes in each cell. A user can add text in these text boxes and hence should behave as a Data Grid.

We found that ARIA-Practices-1.1 defines the following keyboard interaction with Data Grids - 

![image](https://cloud.githubusercontent.com/assets/13657916/23165433/142d1d54-f862-11e6-9fcd-fd968a02ea80.png)

In this we are specifically aiming to achieve functionality for Up/Down and Control+Home/End keys. I am unable to find a way to achieve this in the DetailsList control.

Requesting for these functionalities in the control.

Thanks,
Satish

**Comment by dzearing** [comment]
This is not really supported by the DetailsList architecture, which uses FocusZones to manage rows and to make the inner cells accessible. Doing what you want requires a rethinking of both focus management and rendering. We have plenty of feature asks for this component though and need it ourselves, but this requires a grid more than a row list.
